tests/test_slurm.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import os
+
+def test_slurm(rootdir):
+    sp = subprocess.run(['python',
+     '-m',
+     'wannierberri.slurm',
+     '--exp-name',
+     'my_first_job',
+     '--num-nodes',
+     '4',
+     '--partition',
+     'express',
+     '--command',
+     "'python -u wb-example.py'",
+     '--num-gpus=3',
+     '--sleep-head',
+     '123.45',
+     '--sleep-worker=67',
+     '--no-submit']
+    , capture_output=True )
```

Comment:
`capture_output` was introduced in python 3.7. https://docs.python.org/3/library/subprocess.html. To use this feature, we need to update `python_requires` in `setup.py`.

It would be good to add tests for all supported python versions. To do so, we need to modify `.github/workflows/ci.yml`, line `python-version: [3.8]` to e.g. `python-version: [3.7, 3.8]`.

Reply:
Done, checks are passed both for 3.7 and 3.8

Reply:
done! tests are passing for 3.7 and 3.8